### DISCUSSION TITLE
500 Error uploading file in production but not dev

### DISCUSSION THREAD

**Discussion by dmeinke-etc** [adonisjs/core#3787] [Help]
I'm using the Drive plugin to upload files and when in production mode it just throws a 500 error. But when I switch the production code base to run in dev with npm run dev it will show all my console.log entries and it actually works. 

How would I trouble shoot this in production where I can see some console.log entries or where should I start looking? 

Very hard to fix with no error messages.

**Comment by thetutlage** [comment]
You can console log the `error.original` property. Something like this

```ts
try {
  await Drive.<someAction>()
} catch (error) {
  console.log(error.original)
}
```
